larva/src/main/java/org/frankframework/larva/LarvaTool.java

```diff
@@ -670,7 +670,7 @@ public void wrongPipelineMessage(String stepDisplayName, String message, String
 		writeLog("<pre id='" + diffBoxId + "' class='diffBox'></pre>", method, false);
 		writeLog("</div>", method, false);
 
-		if (config.getLogLevel().shouldLog(LarvaLogLevel.SCENARIO_PASSED_FAILED)) {
+		if (config.getLogLevel() == LarvaLogLevel.SCENARIO_PASSED_FAILED) {
```

Comment:
Hiermee vergelijk je de reference en niet per se de inhoud.
```suggestion
		if (LarvaLogLevel.SCENARIO_PASSED_FAILED.equals(config.getLogLevel())) {
```

Reply:
Dit is een enum :)

Reply:
Oeps